commit 537a836dd6db384d53b52eb457a7d257c440217f
Author: Chris Wilson <chris@chris-wilson.co.uk>
Date:   Fri Apr 1 07:10:25 2011 +0100

    946GZ is a 965G!
    
    Sales & Marketing score another victory in confusing me.
    
    Bugzila: https://bugs.freedesktop.org/show_bug.cgi?id=35854
    Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>

diff --git a/src/intel_module.c b/src/intel_module.c
index f6cc6d4..8416544 100644
--- a/src/intel_module.c
+++ b/src/intel_module.c
@@ -323,7 +323,7 @@ void intel_detect_chipset(ScrnInfoPtr scrn,
 	break;
     case PCI_CHIP_I946_GZ:
 	chipset->name = "946GZ";
-	chipset->info = &intel_i915_info;
+	chipset->info = &intel_i965_info;
 	break;
     case PCI_CHIP_I965_GM:
 	chipset->name = "965GM";
